### PR TITLE
Cleanup: records.config cleanup for references and comments.

### PR DESCRIPTION
**Pull request by brbzull0**
Some more clean up around the records.config

### CODE REVIEW

File: ci/jenkins/bin/cache-tests.sh [hunk 1]
```diff
@@ -58,9 +57,10 @@ ${PREFIX}/bin/trafficserver stop
 
 # Now run it again, maybe, with the tweaked configs
 if [ "" != "$TWEAK" ]; then
-#    echo "CONFIG proxy.config.http.cache.required_headers INT 1" >> $RECORDS
-    echo "CONFIG proxy.config.http.negative_caching_enabled INT 1" >> $RECORDS
     ${PREFIX}/bin/trafficserver start
+    sleep 5
+    ${PREFIX}/bin/traffic_ctl config set proxy.config.http.negative_caching_enabled 1
+    sleep 5
```
Comment (cmcfarlen) [inline ci/jenkins/bin/cache-tests.sh:63]: This sleep here isn't very exciting.  If we installed `yq` to the ci image you could do: `yq -i '.ts.http.negative_caching_enabled=1' $RECORDS` before starting traffic server. Or if traffic_ctl could do the out-of-band edit.

Comment (brbzull0) [reply ci/jenkins/bin/cache-tests.sh:63]: True. 

traffic_ctl will take care of that. I’ll link this to the other pr tomorrow.  And update this line . 
thanks

File: plugins/experimental/url_sig/README [hunk 2]
```diff
@@ -32,7 +32,7 @@ Quick install:
 
 
 Edge cache debugging
-	To enable the TSDebug verbose logging, change records.config to have:
+	To enable the TSDebug verbose logging, change records.yaml to have:
 
 		CONFIG proxy.config.diags.debug.enabled INT 1
```
Comment (randall) [inline plugins/experimental/url_sig/README:37]: should these lines be converted to yaml?

**Comment by zwoop** [comment]
Are you going to address Chris' concerns, or that for another PR? Your comment isn't clear. :).

**Comment by brbzull0** [comment]
> Are you going to address Chris' concerns, or that for another PR? Your comment isn't clear. :).

Yes, different PR.

**Pull request merged**